cluster/gce/manifests/etcd.manifest

```diff
@@ -64,13 +64,17 @@
       }
     ],
     "livenessProbe": {
-      "httpGet": {
-        "host": "127.0.0.1",
-        "port": {{ etcd_livenessprobe_port }},
-        "path": "/health"
+      "exec": {
+        "command": [
+          "/bin/sh",
+          "-c",
+          "exec /usr/local/bin/etcdctl --endpoints=127.0.0.1:{{ port }} {{ etcdctl_cerds }} --command-timeout=30s endpoint health"
+        ]
       },
       "initialDelaySeconds": {{ liveness_probe_initial_delay }},
-      "timeoutSeconds": 15
+      "timeoutSeconds": 30,
+      "periodSeconds": 30,
+      "failureThreshold": 5
```

Comment:
5*30s is pretty long. I think either period&timeout  or theshold should be lower...

Reply:
Let's find a correct timeout first, then we will adjust the threshold or period.

Reply:
I will reduce period to 15 to match timeout and will keep threshold set to 5 which should translate to 75 seconds. Is it OK to you?